static/admin/config.yml

```diff
@@ -313,42 +313,43 @@ collections:
                                 [
                                     {
                                         label: "NANOG",
-                                        name: "nanog",
+                                        name: "NANOG",
```

Comment:
I changed these when I was making the enum: gene symbols are already a unique identifier so it seemed easiest to me to keep the formatting the same for both display and id, but happy to change it back